docs/fileset/README.md

```diff
@@ -128,15 +163,32 @@ If you do not have real-time available for your project, you can do a request to
 It's possible to query a fileset in the same way as datasets. The only difference is when performing an insert query, you need to use the `upload()` method instead.
 All fileset records will have fields such as name, size, URL, etc. These fields, as well as any custom fields, can be used for select queries. Please keep in mind that these default fields are immutable.
 
-<CodeSwitcher :languages="{js:'JavaScript',bash:'cURL'}">
+<CodeSwitcher :languages="{js:'JavaScript',py:'Python',bash:'cURL'}">
+<template v-slot:py>
+
+``` py
+...
```

Comment:
I would suggest to use full example of code everytime, because some users can open this part of documentation by directly link. And in this case they need to find another part of code.

Reply:
same here